src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -550,6 +556,10 @@ public static class Items {
         public static final TagKey<Item> ORES_IN_GROUND_STONE = tag("ores_in_ground/stone");
         public static final TagKey<Item> PLAYER_WORKSTATIONS_CRAFTING_TABLES = tag("player_workstations/crafting_tables");
         public static final TagKey<Item> PLAYER_WORKSTATIONS_FURNACES = tag("player_workstations/furnaces");
+        /**
+         * All kinds of potion items that can hold various effects including splash and lingering potions from vanilla.
+         */
+        public static final TagKey<Item> POTIONS = tag("potions");
```

Comment:
I feel like this tag needs more javadoc. Is this any potions, or just potions that convert 1 to 1 with vanilla? e.g. I have a potion bucket, should it be added even though it contains 4x the potion? Or is it left out?

Does it include potions not stored in a glass bottle? or is it glass bottle specific?

Reply:
Some Fabric folks said to subtag it. So there's `c:potions` for your bucket and `c:potions/bottled` for the bottle variations

Reply:
This logic feels wrong to me, if we are sub tagging `c:potions/bottled` shouldn't that imply the potential existence of `c:potions/bucketed` and maybe tags for other container types?

Rather than `c:potions` directly containing all container types other than bottles. Just feels wrong and backwards from how all other sub tags are treated.

Reply:
Wdym? c:potions still exist. And would contain /bottled or /bucketed or whatever other mods do. If not subtagged. Then mods looking for bottle variations will get buckets and other containers they do not want and no way to filter them out. 

Like glass bottle can become potion, lingering potion, and splash potion. All still bottles containing a potion effect but have different behaviors. A mod may want to target just those bottle variations. But how could they do that if c:potions has buckets and bowls?

Reply:
I might have worded that badly just feels wrong that we are providing /bottled without also providing another common use case /bucketed for potion buckets.

I mean is that not the point of this whole convention tag thing, provide all the common tag usages to unify things rather than mods providing their own formats.

Reply:
Devil’s advocate, In this case /bottled is an example using the vanilla bottles. What would stop mods from adding /bucketed? Like how we don’t provide ores/aluminum as we expect the existing ores tags with vanilla ores to provide an example for mods to know they can add their own subtag.

Reply:
In the case of ores its fine not providing all the common tags as A) theres too many different ore variations and B) you cant really get the format wrong its `ores/<oreName>`.

Where as here the format could differ between mods `potions/<containerType>` is just too broad and could lead to different potion bucket tags, which defeats the purpose of convention tags imo.

bucketed vs bucketable vs buckets vs bucket - I could see mods adding any one of these for potion buckets.

But to be honest this is a bit of nit pick, not providing the bucketed tag is fine just something that jumped out at me and felt off from everything else.

Reply:
I can mention it in the javadoc that it would be bucketed as a bucket example

Reply:
> Where as here the format could differ between mods `potions/<containerType>` is just too broad and could lead to different potion bucket tags, which defeats the purpose of convention tags imo.

If only there was a way for multiple mod devs to talk to each other to ensure their conventions were consistent outside of making PRs. If only...

Seriously, we have been doing convention tags for years longer than they have existed in Forge or Neo, the lack of it being in the modloader just means you use whatever convention makes sense and change if someone else ended up using a different convention, thats what issue trackers on mods are for.

For something like buckets, it just sounds dumb that Neo has to define the tag for it since its a niche thing, I honestly doubt any mod besides Tinkers is doing drinkable potion buckets right now so I don't see why its an issue. If it becomes one, me and the other dev adding potion buckets will agree on a tag by talking to each other like open source mod devs.